### ISSUE TITLE
Schema mismatch on fetch node

### ISSUE THREAD

**Issue by paullewis** [breadboard-ai/breadboard#2498]
According to our schema a port's default value should be [`string|undefined`](https://github.com/breadboard-ai/breadboard/blob/main/packages/breadboard/src/types.ts#L133)

However in the fetch component we set the default for the [`headers` property to be an object](https://github.com/breadboard-ai/breadboard/blob/main/packages/core-kit/src/nodes/fetch.ts#L98)

Not sure which is more correct, but the net result is that the Visual Editor is expecting `string|undefined` but in this case gets `{}` which then gets serialized to `[object Object]`

**Comment by dglazkov** [comment]
This is my bug. I defined `default` as "string", because that's what `examples` type is, but actually, JSON schema doesn't have a strong opinion here, and having `default` be of the same type of as its port seems more logical.

The fix here is likely to change `default` to be of `unknown` type and rely on Build API to infer the proper type. Right?

**Comment by aomarks** [comment]
Headers should be `Record<string, string>` in TypeScript, or

```json
{
  "type": "object",
  "additionalProperties": {
  }
}
```

**Comment by aomarks** [comment]
> The fix here is likely to change `default` to be of `unknown` type and rely on Build API to infer the proper type. Right?

Yeah, I agree the default type for a Breadboard port should be any JSON value which is:

Breadboard Type Expression:
```ts
"unknown";
```

TypeScript:
```ts
type JsonSerializable =
  | string
  | number
  | boolean
  | null
  | Array<JsonSerializable>
  | { [K: string]: JsonSerializable };
```

JSON Schema:
```json
{}
```

OR identically:
```json
{"type": ["array", "boolean", "null", "number", "object", "string"]}
```
